addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -321,7 +321,12 @@ def _rtc_join_call(self, store: Store = None, check_rtc_session_ids=None, camera
         if len(self.channel_id.rtc_session_ids) == 1:
             body = Markup('<div data-oe-type="call" class="o_mail_notification"></div>')
             message = self.channel_id.message_post(body=body, message_type="notification")
-            self.channel_id.last_call_message_id = message
+            # sudo - discuss.call.history: can create call history when call is created.
+            self.env["discuss.call.history"].sudo().create(
+                {"channel_id": self.channel_id.id, "start_call_message_id": message.id}
+            )
+            self.channel_id._bus_send_store(message, ["call_history_ids"])
```

Comment:
It would be best if we could move this to the create of session too, but it's less problematic than for unlink as starting a call from just create is not really sufficient (as opposed to calling unlink which fully removes the participant).